list-module-callers/src/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as core from "@actions/core";
+import * as fs from "fs";
+import * as path from "path";
+import * as child_process from "child_process";
+import {buildModuleToCallers, resolveRelativeCallTree} from "./lib";
+
+try {
+    const configFiles = fs
+        .readFileSync(core.getInput("config_files"), "utf8")
+        .split("\n");
+    const moduleFiles = fs
+        .readFileSync(core.getInput("module_files"), "utf8")
+        .split("\n");
+
+    const rawModuleCalls: Record<string, Array<string>> = {};
+
+    const allTerraformFiles = Array.from([...configFiles, ...moduleFiles]);
+    allTerraformFiles.forEach(tfFile => {
+        if (tfFile == "") {
+            return;
+        }
+
+        const tfDir = path.dirname(tfFile);
+        const inspection = JSON.parse(child_process.execSync(`/home/runner/go/bin/terraform-config-inspect --json ${tfDir}`).toString("utf-8"));
```

Comment:
The path to terraform-config-inspect depends on the environment.

Reply:
Added GOBIN/GOPATH detection.
https://github.com/suzuki-shunsuke/tfaction/pull/1528/commits/3acf082d442915aba5649f91307f54a88cbafffb